js-project-guidelines.md

```diff
@@ -179,17 +183,24 @@ Inside the package.json, the main file exported is the one from the auto-generat
 
 Here you can find samples for [Travis](examples/travis.example.yml) and [circle](examples/circle.example.yml).
 
+We also use [coveralls.io](https://coveralls.io/) to automatically publish coverage reports. This is done from travis using
+
+```yml
+script:
+  - npm run coverage
+after_success:
+  - npm run coverage-publish
+```
+
 ##### `.gitignore`
 
-To avoid checking in the dist and lib folders, the `.gitignore` file should at least contain:
+To avoid checking in unwanted files, the `.gitignore` file should follow
+this [example](examples/.gitignore).
```

Comment:
This document isn't only for large js-ipfs-* style projects. It is also for small javascript projects that frankly do not need all of these things. The .gitignore here seems excessively bloated for some smaller repos. For example, [dnslink-deploy](https://github.com/ipfs/dnslink-deploy) is a JS repo that doesn't need this.